src/orion/core/worker/transformer.py

```diff
@@ -239,6 +244,33 @@ def domain_type(self):
         return self.transformer.target_type
 
 
+class Precision(Transformer):
+    """Round real numbers to requested precision."""
+
+    domain_type = 'real'
+    target_type = 'real'
+
+    def __init__(self, precision=4):
+        """Initialize a precision transformation. Precision must be a non-negative integer."""
```

Comment:
You can drop the docstring in `__init__`, we document arguments in the class docstring. (any documented `__init__` you see were mistakes we did not correct yet). This is following numpy's standard: https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring